ci-operator/step-registry/openshift/upgrade/qe/test/cert-rotation/openshift-upgrade-qe-test-cert-rotation-commands.sh

```diff
@@ -83,3 +83,56 @@ oc adm ocp-certificates remove-old-trust -n openshift-kube-apiserver-operator co
 oc adm wait-for-stable-cluster
 oc adm reboot-machine-config-pool mcp/worker mcp/master
 oc adm wait-for-node-reboot nodes --all
+
+# Regenerating the Internal CA for Ingress
+proxyKey="temProxy.key"
+proxyCert="temProxy.pem"
+cusIngressKey="wilcard.key"
+cusIngressCsr="wildcard.csr"
+cusIngressCert="wildcard.pem"
+keyLength=4096
+baseDomin=$(oc get dns.config cluster -o=jsonpath='{.spec.baseDomain}')
+defaultIngressDomin=$(oc get ingresscontroller default  -o=jsonpath='{.status.domain}' -n openshift-ingress-operator)
+currentPath=$(pwd)
+workPath="/tmp/replcertforingress"
+sudo mkdir $workPath; sudo chmod 664 $workPath; cd $workPath
```

Comment:
I hit error `error: error reading /tmp/replcertforingress/temProxy.pem: permission denied` 
please check if chmod related

Reply:
From error /bin/bash: line 99: sudo: command not found in the below review comment， so sudo mkdir $workPath doesn't work, I will remove all sudo commands, thanks.